src/components/RadioGroup/RadioGroup.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { FC, ChangeEvent, FocusEvent } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import RadioInput from './RadioInput/RadioInput';
+import InputValidationMessage from '../InputValidationMessage/InputValidationMessage';
+import styles from './RadioGroup.module.scss';
+
+/**
+ * Allows users to pick a single option from a predefined list of two or more options.
+ * Ideally, it should be used when there are less than 5 options,
+ * otherwise you should consider using SelectInput instead.
+ */
```

Comment:
Actually, this seems like documentation, and since Storybook is so good at managing this, I'm wondering if we should just remove it in the files.

This would have the added benefit of ensuring that the two sources of documentation don't get out-of-step.